core/common/src/main/java/alluxio/util/io/FileUtils.java

```diff
@@ -282,10 +277,12 @@ public static void createStorageDirPath(String path) throws IOException {
    * @param filePath pathname string of the file to create
    */
   public static void createFile(String filePath) throws IOException {
-    File file = new File(filePath);
-    com.google.common.io.Files.createParentDirs(file);
-    if (!file.createNewFile()) {
-      throw new IOException("File already exists " + filePath);
+    Path storagePath = Paths.get(filePath);
+    Files.createDirectories(storagePath.getParent());
+    try {
+      Files.createFile(storagePath);
+    } catch (FileAlreadyExistsException e) {
```

Comment:
why we need this? `FileAlreadyExistsException` is already a subclass of `IOException`